snuba/migrations/groups.py

```diff
@@ -73,6 +50,27 @@ def load_migration(self, migration_id: str) -> Migration:
             raise MigrationDoesNotExist("Invalid migration ID")
 
 
+class ConfigurationLoader(DirectoryLoader):
```

Comment:
I think this should inherit from the base `GroupLoader` not DirectoryLoader which supposed to be a completely separate implementation

Reply:
I did this because otherwise I'd have to copy the `load_migration` function over.